jso/apis/src/main/java/org/teavm/jso/ajax/XMLHttpRequest.java

```diff
@@ -54,6 +57,27 @@ public abstract class XMLHttpRequest implements JSObject {
     @JSProperty("onreadystatechange")
     public abstract void setOnReadyStateChange(ReadyStateChangeHandler handler);
```

Comment:
I would overload this method to take `EventListener` as well.

Reply:
Yes, that would work, thanks for the advice.